ci/do_ci.sh

```diff
@@ -9,29 +9,31 @@ echo "building using ${NUM_CPUS} CPUs"
 
 if [[ "$1" == "bazel.debug" ]]; then
   echo "bazel debug build with tests..."
-  ${BAZEL} build ${BAZEL_BUILD_OPTIONS} //source/exe:envoy-static
-  ${BAZEL} test ${BAZEL_BUILD_OPTIONS} --test_output=all \
-    --cache_test_results=no //test/...
+  cd "${ENVOY_CONSUMER_SRCDIR}"
```

Comment:
Is there value in continuing to build/test just the main project? I can't think of anything off the top of my head but I'm wondering if we are going to miss some type of failure that only shows up when attempting to built the primary project standalone?